dags/glam_fog.py

```diff
@@ -201,23 +187,25 @@
 
             sample_counts = view(task_name=f"{product}__view_sample_counts_v1")
 
+            done = EmptyOperator(task_id=f"{product}_done")
```

Comment:
The only DAG watching this task now is `bqetl_glam_refresh_aggregates` and the task name was changed there accordingly in https://github.com/mozilla/bigquery-etl/pull/6677